### ISSUE TITLE
Update support library to 26.0.0

### ISSUE THREAD

**Issue by debaj** [square/assertj-android#221]
TransportController and TransportMediator classes were removed from support-v4 26.0.0 and their functionality was replaced with MediaControllerCompat. This is a breaking change as apps using the latest support library fail to compile with assertj, even when they are not using any of the above classes.

**Comment by JakeWharton** [comment]
Pull requests are welcome!

**Comment by debaj** [comment]
PR created, please review.

**Comment by jhansche** [comment]
There is a workaround, albeit a hack.  Simply create empty classes matching the removed classes into the `src/test/java/android/support/v4/media/` directory:

```
package android.support.v4.media;

public class TransportController {}
```
```
package android.support.v4.media;

public class TransportMediator {}
```

**Comment by debaj** [comment]
Yeah we are doing a similar thing, we replaced the assertion class without the two missing support classes.

**Comment by JakeWharton** [comment]
Released as 1.2.0
